sisl/viz/_env_vars.py

```diff
@@ -0,0 +1,53 @@
+import os
+
+__all__ = ["ENV_VARS", "register_env_var", "get_env_var"]
+
+ENV_VARS = {}
```

Comment:
This should be named after the module, so something like `SISL_ENVIRONMENT` would be explicit and concise. It refers to a global variable which has *stuff*.

Also, I have some ideas for this, I will create a PR for your branch to move this to a more generic place! :)

Reply:
Yeah I saw at some point that you were using environment variables as well. Good, I leave this to you :+1: